pages/docs/reference/coding-conventions.md

```diff
@@ -726,6 +727,201 @@ foo {
 
 </div>
 
+### Trailing commas
+
+A trailing comma is a comma sign after the last item of the element series:
+
+<div class="sample" markdown="1" theme="idea" mode="kotlin" data-highlight-only>
+
+```kotlin
+class Person(
+    val name: String,
+    val surname: String,
+    val age: Int, // trailing comma
+)
+```
+
+</div>
+
+Using trailing commas has several benefits:
+
+* It makes version-control diffs cleaner: all focus is on changed value.
+* It's easy to add or reorder elements: no need to add or delete the comma sign if you manipulate elements.
+* It simplifies code generation (for example, for object initializers). The last element can also have the comma sign.
+
+Using trailing commas are optional: your code will work without them. But Kotlin style guide encourages the use of trailing commas at the declaration site and leaves it optional for the call site.
+
+To enable trailing commas for the IntelliJ formatter, go to __Settings | Editor | Code Style | Kotlin__, 
+open the __Other__ tab and enable the __Use trailing comma__ option.
+
+Kotlin supports trailing commas in the following cases:
+* [Lists and enumerations](#lists-and-enumerations)
```

Comment:
The support in Lists is the same as in value arguments.

Reply:
Is it ok if I merge them into one chapter?

I mean
`### Lists, enumerations, value arguments`

Reply:
I suggest to have only enumeration chapter and `value arguments` as `Lists` is subset of `value arguments`. `listOf(a, b, c)` is just a function call.

Reply:
To recap.
No more `Lists and enumerations`: there are `Enumerations` and `Value arguments`.